### PR TITLE
Updated Course Migration Command and View

### PR DESCRIPTION
**Pull request by clintonb**
- Command now raises an exception and provides more detail when calls to external APIs fail
- View now specifies settings to be used by the command

XCOM-193

FYI @rlucioni @jimabramson

### CODE REVIEW

**Comment by rlucioni** [comment]
:+1:

File: ecommerce/courses/views.py [hunk 1]
```diff
@@ -38,14 +39,17 @@ def get(self, request, *_args, **_kwargs):
...
+        user = request.user
+        logger.info(msg, user.username, course_ids)
 
         if not course_ids:
             return HttpResponse('No course_ids specified.', status=400)
 
         course_ids = course_ids.split(',')
 
-        call_command('migrate_course', *course_ids, commit=commit, stdout=out, stderr=err)
+        print user.access_token
```
Comment (jimabramson) [inline ecommerce/courses/views.py:50]: is that intentional?

Comment (clintonb) [inline ecommerce/courses/views.py:50]: Most definitely not.

Comment (rlucioni) [inline ecommerce/courses/views.py:50]: Nice save @jimabramson.

**Pull request merged**